internal/test/wmcb/runTest.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+cd wmcb
+
+# Transfer the files and run the unit and e2e tests
+if ! CGO_ENABLED=0 GO111MODULE=on go test -v -run=TestWMCB \
```

Comment:
Is it not possible to build a test binary in the Dockerfile and transfer it? You can the specify the command in the deployment and remove this script.

Reply:
will look into this and update.

Reply:
Please update.

Reply:
sorry, missed this. 
actually i am not sure how that will work. 
you want me to create a binary using `-c` and then use this binary to run the tests on the pod? 
in that case, i am not sure if it will transfer the required files to the VM

Reply:
Yes create the binary like other test binaries. And you will need to transfer the binary as part of building the test container image. But I am willing to let it go at this point.

Reply:
This too should have been brought up during the standup.